tests/pytests/functional/transport/tcp/test_message_client.py

```diff
@@ -69,7 +80,7 @@ def handler(msg):
         received.append(msg)
 
     client.on_recv(handler)
-
+    await asyncio.sleep(0.03)
```

Comment:
why 0.3 magic number

Reply:
The test seems to pass without this sleep. Hopefully changing this won't cause it to become flaky.